movement/template/template.c

```diff
@@ -76,7 +73,12 @@ bool <#watch_face_name#>_face_loop(movement_event_t event, movement_settings_t *
             // watch_start_tick_animation(500);
             break;
         default:
-            break;
+            // Movement's default loop handler will step in for any cases you don't handle above:
+            // * EVENT_LIGHT_BUTTON_DOWN lights the LED
+            // * EVENT_ALARM_BUTTON_UP moves to the next watch face in the list
+            // * EVENT_MODE_LONG_PRESS returns to the first watch face in the list
+            // You can override any of these behaviors by adding a case for these events to this switch statement.
+            return movement_default_loop_handler(event, settings);
     }
 
     // return true if the watch can enter standby mode. If you are PWM'ing an LED or buzzing the buzzer here,
```

Comment:
This comment is not really accurate, AFAIK, since the movement code handles these cases automatically.

Reply:
There is a typo here (`EVENT_ALARM_BUTTON_UP` should be `EVENT_MODE_BUTTON_UP `) but otherwise I think it is accurate? 

In context, this comment will appear in `my_watch_face.c`. If in the switch statement in that file, I define a case for `EVENT_MODE_LONG_PRESS`, the `default:` case will never be reached, and I will have overridden the behavior; the Movement code will not handle `EVENT_MODE_LONG_PRESS`.

Reply:
Sorry, my comment was, err, about the subsequent comment (not the one you added). I've tried to explain better in my other response.